tests/dummy/app/components/when-route-painted.js

```diff
@@ -4,11 +4,11 @@ import { whenRoutePainted } from 'ember-app-scheduler';
 
 export default Component.extend({
   layout,
-  whenRoutePainted: false,
 
   init() {
     this._super(...arguments);
-
+    this.set('whenRoutePainted', false);
+    debugger;
```

Comment:
Let's take this out

Reply:
Already done. You're looking at an old commit.